pymeasure/instruments/proterial/rod4.py

```diff
@@ -109,36 +95,28 @@ class ROD4(Instrument):
         print(rod4.ch_4.actual_flow)    # Prints Channel 4 actual MFC flow in %
 
     """
-    ch_1 = Instrument.ChannelCreator(ROD4Channel, 1)
-    ch_2 = Instrument.ChannelCreator(ROD4Channel, 2)
-    ch_3 = Instrument.ChannelCreator(ROD4Channel, 3)
-    ch_4 = Instrument.ChannelCreator(ROD4Channel, 4)
 
     def __init__(self, adapter, name="ROD-4 MFC Controller", **kwargs):
         super().__init__(
-            adapter, name, read_termination='\r', write_termination='\r', **kwargs
+            adapter, name, read_termination='\r', write_termination='\r',
+            includeSCPI=False, **kwargs
         )
-        self._keyboard = None
+
+    ch_1 = Instrument.ChannelCreator(ROD4Channel, 1)
+    ch_2 = Instrument.ChannelCreator(ROD4Channel, 2)
+    ch_3 = Instrument.ChannelCreator(ROD4Channel, 3)
+    ch_4 = Instrument.ChannelCreator(ROD4Channel, 4)
 
     version = Instrument.measurement(
-        "\x0200RVN",
+        "\x0201RVN",
         """Get the version and series number. Returns x.xx<TAB>S/N """
     )
 
-    @property
-    def keyboard(self):
-        """Control the front keyboard lock status.
-        Valid options are unlocked or locked. """
-        if self._keyboard is None:
-            log.info('ROD-4 keyboard status has not been set')
-        return self._keyboard
-
-    @keyboard.setter
-    def keyboard(self, status):
-        values = {'unlocked': 0, 'locked': 1}
-        if isinstance(status, str) and status.casefold() in values:
-            status = status.casefold()
-            self.write("\x0200SKO%d" % values[status])
-            self._keyboard = values[status]
-        else:
-            log.info('Status must be unlocked or locked.')
+    keyboard = Instrument.setting(
```

Comment:
Hey,
What about `keyboard_locked`? That would make it more easy to use: `keyboard_locked = True` without thinking about the strings and what True/False mean.

Reply:
Thanks for the suggestion, I've changed it.